src/main/java/uk/gov/hmcts/divorce/document/print/model/Print.java

```diff
@@ -11,19 +12,22 @@ public class Print {
     private final String caseRef;
     private final String letterType;
     private final List<String> recipients;
+    private final Boolean isInternational;
```

Comment:
I think from the code setting it that we are expecting this to be boolean, ie not null just either true or false

Reply:
Correct.  The constructor should be passed a YesOrNo, which it will convert into a Boolean.  However it is possible for the constructor to receive a null value in place of the YesOrNo, in which case it will treat that value as if it is False.

Reply:
I just meant it could be boolean not Boolean is all